### PR TITLE
DE-756 | `optimizeTopK` View [3.12]

### PR DESCRIPTION
**Pull request by aMahanna**
Add test case to validate that `optimizeTopK` is now a supported property in ArangoSearch Views

### CODE REVIEW

File: tests/test_view.py [hunk 1]
```diff
@@ -124,6 +124,19 @@ def test_view_management(db, bad_db, col, cluster):
...
+            view_name,
+            view_type,
+            properties={
+                "links": {col.name: {"fields": {}}},
+                "optimizeTopK": [
+                    "BM25(@doc) DESC",
+                ],
+            },
+        )
+        assert "optimizeTopK" in res
```
Comment (aMahanna) [inline tests/test_view.py:138]: this is failing

Comment (aMahanna) [reply tests/test_view.py:138]: example CI failure: https://app.circleci.com/pipelines/github/arangodb/python-arango/350/workflows/5c928ec3-5bb4-4371-be55-266fedb562d7/jobs/8415

**Review by alexbakharew — CHANGES_REQUESTED** [review]
Hi! `optimizeTopK` was introduced only for 3.12. That is why it is failing for 3.11

**Comment by aMahanna** [comment]
@alexbakharew thank you! resolved

**Review by apetenchea — APPROVED** [review]
LGTM

**Pull request merged**